Touch up typos in June 2024 meeting notes

TomSweeneyRedHat opened the pull request:
I had a bad url in links to some of the noted projects, and some md formating issues that I've touched up.

vrothberg reviewed (commented):
LGTM

lsm5 commented:
err, I guess it's libpod in the official debian repos. I'll change the package name there.

rhatdan commented:
LGTM

EvanCarroll commented:
This is not what I would want or what I would suggest.

* Many debian users use testing. Testing packages can still lag weeks or months behind upstream. For example, Debian testing ships Buildah 1.15. The current version is 1.18.
* By not shipping packages on Debian testing you're missing a huge market that can report bugs, and is more likely to want experimental packages.

I just wanted the packages for Debian Testing to `Conflicts` with the upstream packages provided by debian in the actual rules (debian spec file). Just call them something else and make them conflict in the spec file, then installing one removes the other and everything works.

Either that, which is ideal, or just add the command to `apt-get --purge remove` on all of the debs you conflict with in your documentation.

TomSweeneyRedHat committed:
Touch up typos in June 2024 meeting notes

I had a bad url in links to some of the noted projects, and some md
formating issues that I've touched up.

Signed-off-by: tomsweeneyredhat <tsweeney@redhat.com>

TomSweeneyRedHat commented:
Going to smash this through to clear up typos.
